includes/forum-profile.php

```diff
@@ -397,8 +398,18 @@ public function myProfileLink() {
                 // Get and build profile link.
                 $profileLink = $this->getProfileLink($currentUserObject);
 
-                echo '<a class="profile-link" href="'.$profileLink.'">'.__('Profile', 'asgaros-forum').'</a>';
+            } else {
+                $profileLink = 'profile link';
```

Comment:
Wird dieser Part benötigt? Zwei Strings als Link machen irgendwie keinen Sinn.

Reply:
In der Bedingung ist es natürlich nicht ganz so sinnvoll, aber die da die variable bei der Rückgabe verwendet wird sollte sie schon definiert werden. Ich habe die jetzt oberhalb definiert.